### ISSUE TITLE
[CAIA Intake] AE Profile : MHV Appointment Reminders on VA.gov with PHI/PII 

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#85845]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to request collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

## About your team

- **Team name**: AE Profile
- **OCTO product owner:** Chante Lantos-Swett
- **Product name:** Profile - Notification Settings
- **Product manager:**  Travis Cahill
- **Slack channel:** accountexp-authexp
- **Dedicated content writer on your team (if you have one):** `Insert name and/or GH handle`
- **Dedicated a11y specialist on your team (if you have one):** `Insert name and/or GH handle`
 
## About your initiative
MHV notification settings/preferences are gradually migrating to VA.gov. We've worked with CAIA for the current verbiage for the health care section but we've been asked to incorporate additional preferences to track if a veteran has opted in to receiving appointment reminders that include PHI/PII. 

### Current page: 

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/7a9a781e-5d24-40c2-aff3-a18dda79228f)

### Additional content we need help figuring out: 

**Health care**

Health appointment reminders 
:black_square_button: Notify me by email
>    :black_square_button: Include personal health information in unsecure email message

:black_square_button: Notify me by text
>    :black_square_button: Include personal health information in unsecure text message 

### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

Hoping to get content guidance on these additional preferences. 

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

`Enter your description here`

### Will you release this new product incrementally (for example, release to 25% of users to start)?

- [ ]  Yes
- [x]  No
 
 **Note** If you checked yes: Incremental launches require dynamic content. You'll need to ask your developer to build a React widget and provide CAIA with the ID code. CAIA will provide widget content and add the widget to your product’s static page. Please refer to this [GitHub page](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md) for more information.
 
- [x]  I acknowledge that if I change the launch to be incremental, or change it from incremental to 100%, I must notify CAIA of this change as these additional steps will impact deadlines.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [x] Staging (our original initiative was cleared for launch we haven't yet because MHV hasn't been ready)
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
-  We initially worked through this with [our default send story 57409](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57409)

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Supporting artifact <name>](https://www.figma.com/design/e6JEtrwZCInKk9SjZktx2T/Profile---Notification-Settings?node-id=517-3723&t=IcMyzp4O8zjBUBwc-1) `desktop - Appointment Reminders PHI/PII Option 3`

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Naomi Eke`, and we’ll then acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together. 
[slack thread where I linked to](https://dsva.slack.com/archives/C01K37HRUAH/p1718224154276019)
- [ ] Review the remaining next steps in the **VFS team tasks** section here.
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well: [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work: [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next: [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).

**Comment by laurwill** [comment]
Hi @mtcA6 , update here: I sent the proposed email content for privacy review ([ticket here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/58378)), but we haven't yet confirmed what information we'll show in the PHI version. I think we'll want to explain what personal info we'll include above the option to choose PHI. 

Something like: 
You can opt in to receive appointment reminders that include your personal health information, like the name of your VA facility and clinic. If you don't opt in, we won't include your personal health information in these reminders. 

That's too long, but you get the idea. I'm also tracking [updates needed to our VEText page](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87286) to list all the types of PHI that might appear in the PHI version of text reminders. 

I'm OOO next week, but can pick this up on 7/8!

**Comment by mtcA6** [comment]
@laurwill VA Profile needs the naming convention ideally by Tomorrow 7/17/2024 so they can add the appropriate text into their tables are you able to do that? 

cc: @clantosswett

**Comment by mtcA6** [comment]
@laurwill scratch that! We don't need it as urgently as tomorrow, thought we needed it for VA Profile to move forward but it turns out this isn't blocking them. But we will need it in a few sprints so we can support the added functionality they're building.

**Comment by laurwill** [comment]
Hey @mtcA6 , haven't heard back from privacy yet but pinging them today. I think next few sprints should be doable.

Also wanted to connect you with @jamieramsay , who just explained to me that VEText is launching at Oracle facilities this month, but Veterans at those facilities won't be able to manage their VEText notification preferences through profile. 

What does profile currently show Veterans registered at Oracle facilities on this notifications screen? Can they currently manage their preferences here, or do we direct them to an Oracle system for that?

**Comment by JamieRamsay** [comment]
thank you @laurwill hello @mtcA6 . So, it's a little nuanced. They will be able to manage their preferences in VA profile. However, to begin receiving text messages (if they were not previously receiving text messages through the former service Televox) they will need to also confirm with their provider that their preferred contact method in the Revenue Cycle is set to text messages. [This chart](https://github.com/department-of-veterans-affairs/vetext/wiki/VEText-2.0-Message-Opt%E2%80%90in-Rules) may help explain.

**Comment by JamieRamsay** [comment]
Thank you @mtcA6 ! For our part, it does not seem like we need anything to change/update on VA Profile at this point. Veterans can still make the changes in VA Profile per their preference. They may just need to confirm settings in Rev Cycle with their provider for text message appointment reminders.

**Comment by mtcA6** [comment]
@JamieRamsay how would they know to do that?

**Comment by JamieRamsay** [comment]
In national comms we are recommending all Veterans and Federal EHR sites "check with their provider" to confirm they are set up correctly to receive text messages. Locally, these facilities will be communicating with fliers/info as well as have processes in place for MSAs to check/confirm the RevCycle settings for preferred contact method whenever a Veteran comes in.

**Comment by mtcA6** [comment]
Handful of questions: 

- That process covers text, does it also relate to notifications via email? Or is there not a need for RevCycle to have this information when emails are sent? 
- How did MHV handle this in their notifications process?? Was there a true-up between MHV and RevCycle? 
- I thought VEText was sending the text messages and they plug into VA Profile, has this additional layer of RevCycle be discussed with VA Notify / VEText? 
- Are there plans to create a true up between RevCycle and VA Profile which would update VA.gov Profile?

**Comment by JamieRamsay** [comment]
VEText Federal EHR is not currently handling email messages. In the future, VEText will also email, but currently it is only handling text messages.
I am not sure but can direct you to a my colleague who deals more with the Oracle Health side
This additional layer was part of [the SOP]([Oracle Health Millennium VEText SOP JTL edits 06MAY24](https://dvagov.sharepoint.com/:b:/r/sites/vhavetext/FederalEHR/Shared%20Documents/SOPs/Oracle%20Health%20Millennium%20VEText%20SOP%20%20JTL%20edits%2006MAY24.pdf?csf=1&web=1&e=oBrP2H)) 
I am happy to jump on a call to clarify some these things if that helps.

**Comment by mtcA6** [comment]
> In national comms we are recommending all Veterans and Federal EHR sites "check with their provider" to confirm they are set up correctly to receive text messages. Locally, these facilities will be communicating with fliers/info as well as have processes in place for MSAs to check/confirm the RevCycle settings for preferred contact method whenever a Veteran comes in.

Is there a way for us to programmatically check this to reduce the burden? 

I'm struggling to connect all of the dots between the VA.gov profile and Oracle Health / RevCycle. It's two dots, but I'm assuming not all facilities are using Oracle health some vets may go to multiple facilities and they'd have to check w/each facility which is now multiplying the number of dots and places to cross reference. When veterans go into the VA.gov/Profile/Notifications I feel like when they check the box "I want to receive appointment reminders with PHI (naming convention to be clarified by this ticket)" they're going to expect it to just work.

**Comment by JamieRamsay** [comment]
@mtcA6 Just today we got word on an update to the SOP which should simplify connecting the dots. 
VEText will be pulling preferences from VA Profile exclusively.

**Comment by mtcA6** [comment]
@laurwill checking in here. can you opine on the copy? 

---
# Initial submission: 

### Health appointment reminders 
:black_square_button: Notify me by email
>    :black_square_button: Include personal health information in unsecure email message

:black_square_button: Notify me by text
>    :black_square_button: Include personal health information in unsecure text messag

---

# Thoughts on the below? 

Do we need to add a bit more? I actually don't think added copy is bad


### Your health care
#### Health appointment reminders 
You can opt in to receive appointment reminders with or without personal health information. In order to include information like the name of your VA facility and clinic you need to grant us permission. 

:black_square_button: Notify me by email
>    :black_square_button: Include personal health information in unsecure email messages

:black_square_button: Notify me by text
>    :black_square_button: Include personal health information in unsecure text messages


bottom of the page will have:
[**Note:** Text messaging and email aren't encrypted forms of communication. Read more about privacy and security for digital notifications](https://www.figma.com/design/e6JEtrwZCInKk9SjZktx2T/Profile---Notification-Settings?node-id=264-3473&t=zLTjHzf5FJQayLZT-4)

---

As some added context, MyChart has something like the following: 
![image](https://github.com/user-attachments/assets/6ad4a058-7520-411b-92a6-06902213f4e9)

**Comment by mtcA6** [comment]
@twoangstroms should have cc'd you on that last message.

**Comment by twoangstroms** [comment]
@laurwill @JamieRamsay Hi Laura and Jamie – recently joined the AE team and have been starting work on some of this. We'd really like y'all to have a look at the linked Figma file above to get feedback on the wording and placement of the radio button group (and the help text). I tried to follow plain language guidelines.

**Comment by twoangstroms** [comment]
Thanks @laurwill @JamieRamsay and @mtcA6 

The content is really helpful! 

As for the UI, that's a complex issue. I'll try to be brief and clear, but lmk if you have questions, of course.

I explored a lot of options in [this Figma file](https://www.figma.com/design/e6JEtrwZCInKk9SjZktx2T/Profile---Notification-Settings?node-id=572-10323&t=nUkYiUDbxIzAtlxN-0) (make sure you're on the "Desktop - Notifications with PHI options" page, see attached image).

I did look at that way of doing it, with what we could call "sub-checkboxes", but that's not implementable for three reasons. The first is that we cannot indent (for a11y) reasons, as this doesn't render in screen readers. The second is that I was told that the UI elements have to reflect a "channel" in the backend database (where "channel" is basically a this or that). And third, we can't create new header-level items in the UI for the same backend reasons, so we can't break email Into one checkbox or radio button group and text into another.

We're hoping to meet with some backend people this week to get more insight on what is and isn't possible, from having some logic in the backend to... I'm not sure what. It's currently a wicked problem, with the Design System and backend constraints.
<img width="242" alt="Screenshot 2024-08-27 at 1 28 04 PM" src="https://github.com/user-attachments/assets/026a3bf1-a9bf-4a8d-98bc-6767d5a8f8f4">
